### DISCUSSION TITLE
Implement that the app will ask you to rerun it as admin when user try enable Top-Most / Shrink modes for admin window

### DISCUSSION THREAD

**Discussion by ruike-kj** [WindowTop/WindowTop-App#224] [Ideas]
微软商店购买的wintop，怎样设置以管理员开机启动
![2022-05-08_111647](https://user-images.githubusercontent.com/78640327/167280200-625dab39-2d3d-4b8a-96b2-bacf314e1089.jpg)

**Comment by gileli121** [comment]
Hi, this is not supported yet in the store version due to the security feature in the OS for Microsoft store apps.
But I can develop a workaround for this.
But the app will not start with an admin privilege in any case.

What the workaround will do is that when you try to set window on top and admin privilege is needed, the app will give you the option to re-run in as admin and then set the window on top, in a single click.

Edit:
It will do it also when you try to Shrink window with admin privilege - workaround also for #236 issue

**Reply by ruike-kj** [reply]
好的，谢谢开发者，请尽快开发这个功能。我非常需要他。
我是说如果的话，我希望能有更好的解决办法来以管理员开机自启

**Reply by DBX31** [reply]
hi

i'm want this too.
thanks
dbx

**Reply by gileli121** [reply]
@ruike-kj 
@DBX31 
Please check again:
https://github.com/BiGilSoft/WindowTop/issues/241
I submitted a beta version with this feature.

Enjoy!

**Comment by gileli121** [comment]
@DBX31 @ruike-kj 
Implemented:
https://github.com/BiGilSoft/WindowTop/releases/tag/v5.14.0

Enjoy!
